extensions/resteasy-reactive/quarkus-resteasy-reactive-common/deployment/src/main/java/io/quarkus/resteasy/reactive/common/deployment/JsonDefaultProducersHandler.java

```diff
@@ -30,6 +30,11 @@
     }
 
     private boolean isJsonCompatibleType(Context context) {
+        if (context.httpMethod() == null) {
```

Comment:
So I guess that this PR is about ignoring the warning for _sub-resource locators_, right? Because if a request method is present then it's called a _sub-resource method_. ;-)

Reply:
Right. I am not clear myself about the terminology... It's just confusing IMHO :)